### PR TITLE
add tests for flow validation with meter

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSingleSwFlowSpec.groovy [hunk 1]
```diff
@@ -152,6 +154,28 @@ class SwitchValidationSingleSwFlowSpec extends BaseSpecification {
...
+            assert rate.expectedValue == newBandwidth.toString()
+            assert rate.actualValue == flow.maximumBandwidth.toString()
+
+            def burst = direction.discrepancies[1]
+            assert burst.field == "meterBurstSize"
+            Long newBurstSize = switchHelper.getExpectedBurst(sw.dpId, newBandwidth)
+            verifyBurstSizeIsCorrect(newBurstSize, burst.expectedValue.toLong())
+            verifyBurstSizeIsCorrect(burstSize, burst.actualValue.toLong())
+
+            direction.flowRulesTotal == 1
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSingleSwFlowSpec.groovy]: assert keywords are missing here and below

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy [hunk 2]
```diff
@@ -175,6 +179,49 @@ class SwitchValidationSpec extends BaseSpecification {
...
+            assert rate.expectedValue == newBandwidth.toString()
+            assert rate.actualValue == flow.maximumBandwidth.toString()
+
+            def burst = direction.discrepancies[1]
+            assert burst.field == "meterBurstSize"
+            Long newBurstSize = switchHelper.getExpectedBurst(srcSwitch.dpId, newBandwidth)
+            verifyBurstSizeIsCorrect(newBurstSize, burst.expectedValue.toLong())
+            verifyBurstSizeIsCorrect(srcSwitchBurstSize, burst.actualValue.toLong())
+
+            direction.flowRulesTotal == 2
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy]: asserts

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy [hunk 3]
```diff
@@ -175,6 +179,49 @@ class SwitchValidationSpec extends BaseSpecification {
...
+            direction.switchRulesTotal == srcSwitchAmountOfRules + 2
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy]: I don't understand why `direction.switchRulesTotal` would show the amount of rules on src switch. Why not total sum of rules on all the flow's switches? This looks incorrect for me

Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSpec.groovy]: you are right
develop branch was fixed and test was fixed too

**Pull request merged**